react/src/components/atoms/buttons/Button/Button.knobs.options.js

```diff
@@ -3,7 +3,7 @@
  */
 export default {
   theme: {
-    '': 'no theme i.e. primary (default)',
+    'c-primary': 'no theme i.e. primary (default)',
```

Comment:
I think we can leave this as empty, i fixed the proptype warning in the previous pr. that seems like the way we handle default value or usage and other props